src/pages/MembersPage/MembersPage.jsx

```diff
@@ -7,10 +7,7 @@ import tableStyles from '../../styles/table';
 
 import Avatar from 'material-ui/Avatar';
 import Button from 'material-ui/Button';
-import List from 'material-ui/List';
-import ListItem from 'material-ui/List/ListItem';
-import ListItemAvatar from 'material-ui/List/ListItemAvatar';
-import ListItemText from 'material-ui/List/ListItemText';
+import List, { ListItem, ListItemAvatar, ListItemText } from 'material-ui/List';
```

Comment:
Does `create-react-app` perform tree shaking on build time to reduce the extra load time caused here?

Reply:
Webpack 2 comes with it but to optimize it better, we might want to add an uglifyer. 
https://webpack.js.org/guides/tree-shaking/